agent/ansible/roles/pbench-repo-install/vars/main.yml

```diff
@@ -0,0 +1,44 @@
+---
+baseurl: "{{ 'http://coprbe.devel.redhat.com/results/ndokos/pbench/rhel-8.dev-$basearch/'
+                 if ansible_distribution == 'RedHat' and ansible_distribution_major_version == '8'
+                 else
+             'https://copr-be.cloud.fedoraproject.org/results/ndokos/pbench/epel-7-$basearch/'
```

Comment:
Can we factor out the `https://copr-be.cloud.fedoraproject.org/results/ndokos` as a prefix in a variable?